app/Console/Command/UpdatePlaceLocality.php

```diff
@@ -113,26 +144,34 @@ protected function execute(InputInterface $input, OutputInterface $output): ?int
                     throw new Exception('Address is incomplete in Json');
                 }
 
-                $command = new UpdateAddress(
-                    $place->getId(),
-                    new Address(
-                        new Street($jsonLd['address']['nl']['streetAddress']),
-                        new PostalCode($jsonLd['address']['nl']['postalCode']),
-                        new Locality($newLocality),
-                        new CountryCode($jsonLd['address']['nl']['addressCountry'])
-                    ),
-                    new Language('nl')
+                $address = new Address(
+                    new Street($jsonLd['address']['nl']['streetAddress']),
+                    new PostalCode($jsonLd['address']['nl']['postalCode']),
+                    new Locality($newLocality),
+                    new CountryCode($jsonLd['address']['nl']['addressCountry'])
                 );
+
+                $command = $itemType->sameAs(ItemType::place()) ? new UpdatePlaceAddress(
```

Comment:
To increase readability, you can make this a method. Once we are on PHP 8.x, you can even return a union, now the type hint of the return type will need to be comment.